docs/services/sensors.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Sensors Service"
+linkTitle: "Sensors"
+weight: 70
+type: "docs"
+description: "The Sensors Service is a builtin service that provides a central interface to all of your robot's sensors."
+tags: ["sensor", "services"]
+icon: "/services/img/icons/sensor.svg"
+images: ["/services/img/icons/sensor.svg"]
+# SME: Cheuk
+---
+
+The Sensors Service is a builtin service that provides a central interface to all of your robot's [sensors](/components/sensor), regardless of the sensor model.
+With it you can obtain readings from multiple of your robot's sensors at once.
+
+## API
+
+The Sensors Service supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[`Sensors`](#sensors) | Returns a list of names of the available sensors.
+[`Readings`](#readings) | Returns a list of sensor readings from a given list of sensors.
+
+{{% alert title="Note" color="note" %}}
+
+The following code examples assume that you have a robot configured with a gripper, and that you add the required code to connect to your robot and import any required packages at the top of your code file.
```

Comment:
```suggestion
The following code examples assume that you have a robot configured with a sensors service, and that you add the required code to connect to your robot and import any required packages at the top of your code file.
```

Reply:
With this suggestion, would "Sensors Service" be capitalized in that context?